### ISSUE TITLE
Beta 0.3.1 - Server Side Loading

### ISSUE THREAD

**Issue by ileleana** [bubbafett5611/CozyCreate#3]
Hello,

So I am trying to manually load this pack onto a server however I am having issues and I'm not that great at understanding the crash logs would you be able to help? I think part of my problem is I don't know which mods are client side and which are server side.  

https://pastebin.com/K5Uswe7N - Crash Report

Ileleana

**Comment by bubbafett5611** [comment]
Absolutely @ileleana, do you mind answering some questions for me?
What Pack version is this?
What version of forge are you using on the server?
Are you using a hosting service? If so, which one?
Is this a custom world?

**Comment by ileleana** [comment]
Hello,
Yes I am hosting it on Akliz.net
it is pack version Beta 0.3.1
Minecraft Forge - Version 47.2.20 for 1.20.1
I hadn't even loaded the world yet as I was just setting it up and it crashed immediately after transporting the config and mod folders over to the new server via Filezilla sFTP

**Comment by bubbafett5611** [comment]
Can you try forge version: forge-1.20.1-47.2.0-server.jar? That is what I have my server loaded on.

**Comment by ileleana** [comment]
I did so and it crashed again:
https://pastebin.com/xGfSJsiY

seems to mention Oculus and Zume?

**Comment by bubbafett5611** [comment]
AH! The client side mods! My server host automatically disables them.

This weekend I will update the pack and start including server files with it that has the correct jar and client side mods disabled.

**You can delete/disable the following mods from your server.**
- BetterF3
- Better Mods Button
- Better Third Person
- Controllable
- Controlling
- Ding
- Embeddium
- Embeddium++
- Enchantment Descriptions
- Entity Culling
- Extreme Sound Muffler
- Farsight
- It's the little things
- Jade Addons
- Light Overlay
- Mouse Tweaks
- Not Enough Animations
- Notes
- Oculus
- Pick Up Notifier
- Simple Discord RPC
- Skin Layers 3D
- Toast Control

Please let me know if this solves your issue.

**Comment by ileleana** [comment]
Thank you! That worked I have a world spawn lol! I'm still learning what is and isn't server-side mods so I appreciate the help, I had a feeling that I hadn't removed all the client-side mods XD. My server host doesn't automatically disable them but I like the way my server host runs the servers so I've not wanted to switch from them
Thank you again!

**Comment by bubbafett5611** [comment]
Absolutely! Please feel free to reach out with any issues and expect an update within the next few days!
